storagenode/gracefulexit/endpoint.go

```diff
@@ -89,40 +89,66 @@ func (e *Endpoint) GetNonExitingSatellites(ctx context.Context, req *pb.GetNonEx
 }
 
 // StartExit updates one or more satellites in the storagenode's database to be gracefully exiting.
-func (e *Endpoint) StartExit(ctx context.Context, req *pb.StartExitRequest) (*pb.StartExitResponse, error) {
-	e.log.Debug("initialize graceful exit: StartExit", zap.Int("satellite count", len(req.NodeIds)))
-	// save satellites info into db
-	resp := &pb.StartExitResponse{}
-	for _, satelliteID := range req.NodeIds {
-		e.log.Debug("initialize graceful exit: StartExit", zap.String("satellite ID", satelliteID.String()))
-
-		status := &pb.StartExitStatus{
-			Success: false,
-		}
-		domain, err := e.trust.GetAddress(ctx, satelliteID)
-		if err != nil {
-			e.log.Debug("initialize graceful exit: StartExit", zap.Error(err))
-			resp.Statuses = append(resp.Statuses, status)
-			continue
-		}
-		status.DomainName = domain
+func (e *Endpoint) StartExit(ctx context.Context, req *pb.StartExitRequest) (*pb.ExitProgress, error) {
+	e.log.Debug("initialize graceful exit: StartExit", zap.String("satellite ID", req.NodeId.String()))
```

Comment:
Should this maybe be `InitiateGracefulExit` since that is how we refer to it in the DB layer?